tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -3447,6 +3471,59 @@ public function userStoreRecordingFileInRoom(string $user, string $file, string
 		$this->assertStatusCode($this->response, $statusCode);
 	}
 
+	/**
+	 * @Then /^read bot ids from OCC$/
+	 */
+	public function readBotIds(): void {
+		$this->invokingTheCommand('talk:bot:list -v --output json');
+		$this->theCommandWasSuccessful();
+		$json = $this->getLastStdOut();
+
+		$botData = json_decode($json, true, 512, JSON_THROW_ON_ERROR);
```

Comment:
Nit: just in case you want to avoid defining unused parameter:
```suggestion
		$botData = json_decode($json, true, flags: JSON_THROW_ON_ERROR);
```

Reply:
Oh nice, works since PHP 8.0 🤩

Reply:
But another EA bug I guess:
![grafik](https://github.com/nextcloud/spreed/assets/213943/0ffc37be-3ca2-452e-98e8-0fb6faabfd4d)